<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LotSelectClass</class>
 <widget class="QDialog" name="LotSelectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LotSelect</string>
  </property>
  <widget class="QTableWidget" name="twLotList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>561</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>561</width>
     <height>21</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="PasswordPushButton" name="pbSelect">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>360</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>403</y>
     <width>561</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="PasswordPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>420</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>J:/検査画面icon/flash_64.PNG</normaloff>J:/検査画面icon/flash_64.PNG</iconset>
   </property>
  </widget>
  <widget class="PasswordPushButton" name="pbAllSelect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>All Select</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leSearchLotName">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>52</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Search Lot Name</string>
   </property>
  </widget>
  <widget class="PasswordPushButton" name="pbSearch">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>45</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PasswordPushButton</class>
   <extends>QPushButton</extends>
   <header>passwordpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
